ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -167,14 +167,16 @@ function NetworkGraphPage() {
                         includePorts,
                         ALWAYS_SHOW_ORCHESTRATOR_COMPONENTS
                     ),
-                    fetchNetworkPolicyGraph(
+                    // fetch the network graph data, including policies, for the inactive graph
+                    fetchNetworkFlowGraph(
                         selectedClusterId,
                         namespacesFromUrl,
                         deploymentsFromUrl,
                         queryToUse,
                         undefined,
                         includePorts,
-                        ALWAYS_SHOW_ORCHESTRATOR_COMPONENTS
+                        ALWAYS_SHOW_ORCHESTRATOR_COMPONENTS,
+                        INCLUDE_POLICIES
```

Comment:
oh do we still to make the other call without the `INCLUDE_POLICIES` flag?

Reply:
Yea, we still do. Do you think it's unnecessary? Is it possible for the nodes/edges information to differ between both calls?

Reply:
If they don't, we can probably just use one call and redo some of the logic for creating the data models